pyblazing/pyblazing/apiv2/context.py

```diff
@@ -1716,6 +1718,13 @@ def create_table(self, table_name, input, **kwargs):
         input : data source for table.
                 cudf.Dataframe, dask_cudf.DataFrame, pandas.DataFrame,
                 filepath for csv, orc, parquet, etc...
+        file_format (optional) : string describing the file format
+                      (e.g. "csv", "orc", "parquet") this field must
+                      only be set if the files do not have an extension.
+        local_files (optional) : boolean, must be set to True if workers
+                      only have access to a subset of the parquet files
```

Comment:
`subset of the parquet files` should be `subset of the files`

Reply:
Fixed.